2018/2018_File_Generation_Features.md

```diff
@@ -0,0 +1,67 @@
+# 2018 File Generation Features
+
+## The FileGenerator Class. 
+The `FileGenerator()` class is located in [`file_generator.py`](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/file_generator.py) and instantiates without any inputs in the function call. The class uses data stored with yaml configuration files in the 
```

Comment:
Start this section with a description of what the FileGenerator class is meant to be used for. This section should include a brief overview of what the class does and why.

Change `and instantiates without any inputs in the function call.` to `can be instantiated without inputs.`

Reply:
Break out the discussion of configurations into a separate paragraph from the overview of the file generator class.

Reply:
Sure! I am going to move up the discussion of classes as well.